peoplesoft/0.3.6/modules/ROOT/pages/index.adoc

```diff
@@ -245,11 +245,11 @@ javac classpath $PS_HOME/class/psjoa.jar *.java
 . Configure your Component Interface Whitelist according to the steps below:
 .. Click *Create Object manually* and click the button next to it.
 +
-image::ps-0006-globalelement-whitelist.png[PS_0006_globalelement_whitelist]
+image::ps-0006-globalelement-allowlist.png[PS_0006_globalelement_allowlist]
```

Comment:
For accessibility, since there is no caption for the images, we should put a friendly description for the alt text, which should be a brief description of what the image shows.

Reply:
Something like "General properties with the Create Object manually option selected"